common/settings.h

```diff
@@ -316,6 +316,11 @@ struct Settings {
   /// If it is not 0 or 1, it must be one of 2, 4, 8, or 16. However, if the
   /// GPU does not support the requested sampling value, MSAA will be disabled.
   uint8_t msaa_samples = 0;
+
+  /// Enable embedder api on the embedder.
+  ///
+  /// This is ignored on the platforms that use the embedder_api by default.
+  bool enable_embedder_api = false;
```

Comment:
Maybe more self-descriptive naming like `enable_embedder_api_for_ios` (or `enable_embedder_api_for_mobile` if wanna re-use for android)

Reply:
I tried to keep it generic so it could be reused by android.
I also wanted to keep it the same name as the command line flag. And I would like the command line flag to be short for easier typing when testing ;p